homeassistant/components/honeywell/__init__.py

```diff
@@ -102,6 +118,7 @@ async def _retry(self) -> bool:
             return False
 
         self.devices = devices
+        update_listener(self._hass, self._config)
```

Comment:
```suggestion
```

Reply:
Probably should reload directly in case additional logic gets added to the update_listener later.  Also `update_listener` is a co-routine